### PR TITLE
First Commit

### PR DESCRIPTION
**Pull request by cs-sujeet-ahirrao**
Implemented Following Actions
- whois_lookup
- ssl_certificates
- dns_lookup

### CODE REVIEW

File: whois-freaks/images/sns_large.png
Comment (cs-prerna-joshi) [inline whois-freaks/images/sns_large.png:1]: need to update the logo.

File: whois-freaks/info.json [hunk 1]
```diff
@@ -0,0 +1,436 @@
+{
+  "name": "whois-freaks",
+  "label": "Whois Freaks",
+  "version": "1.0.0",
+  "description": "",
```
Comment (cs-prerna-joshi) [inline whois-freaks/info.json]: add description.

File: whois-freaks/info.json [hunk 2]
```diff
@@ -0,0 +1,436 @@
...
+  "publisher": "Fortinet",
+  "cs_approved": false,
+  "cs_compatible": false,
+  "category": "Information",
```
Comment (cs-prerna-joshi) [inline whois-freaks/info.json]:
```suggestion
  "category": "Threat Intelligence",
```

File: whois-freaks/info.json [hunk 3]
```diff
@@ -0,0 +1,436 @@
...
+              },
+              {
+                "title": "Page Number",
+                "name": "page",
+                "visible": true,
+                "required": false,
+                "editable": true,
+                "type": "text",
+                "tooltip": "For getting next or desired page whois info",
+                "description": "For getting next or desired page whois info"
```
Comment (cs-prerna-joshi) [inline whois-freaks/info.json]: reformat the description.

File: whois-freaks/operations.py [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+        self.sslVerify = config.get('verify')
+
+    def make_api_call(self, endpoint=None, method='GET', data=None, params=None):
+        try:
+            endpoint = self.server_url + endpoint
+            headers = {'Content-Type': 'application/json', 'Accept': 'application/json',
+                       "Authorization": "Token {}".format(self.api_key)}
+            response = requests.request(method, endpoint, params=params, headers=headers,
+                                        verify=self.sslVerify, data=json.dumps(data))
+            if response.ok:
```
Comment (cs-prerna-joshi) [inline whois-freaks/operations.py:31]: need to handle the response code 404 - Not found. In this case connector step could not fail.

File: whois-freaks/info.json (hunk 1 above)
Comment (cs-sujeet-ahirrao) [reply whois-freaks/info.json]: done

File: whois-freaks/info.json (hunk 2 above)
Comment (cs-sujeet-ahirrao) [reply whois-freaks/info.json]: done

File: whois-freaks/info.json (hunk 3 above)
Comment (cs-sujeet-ahirrao) [reply whois-freaks/info.json]: done

File: whois-freaks/operations.py (hunk 4 above)
Comment (cs-sujeet-ahirrao) [reply whois-freaks/operations.py:31]: done

**Pull request merged**